InnerEye/ML/run_ml.py

```diff
@@ -439,10 +436,11 @@ def run_inference_for_lightning_models(self, checkpoint_paths: List[Path]) -> No
             # Read the data modules before changing the working directory, in case the code relies on relative paths
             data = self.container.get_inference_data_module()
             dataloaders: List[Tuple[DataLoader, ModelExecutionMode]] = []
-            if self.container.perform_validation_and_test_set_inference:
+            if self.container.run_perform_inference(ModelProcessing.DEFAULT, ModelExecutionMode.TEST):
                 dataloaders.append((data.test_dataloader(), ModelExecutionMode.TEST))  # type: ignore
+            if self.container.run_perform_inference(ModelProcessing.DEFAULT, ModelExecutionMode.VAL):
                 dataloaders.append((data.val_dataloader(), ModelExecutionMode.VAL))  # type: ignore
-            if self.container.perform_training_set_inference:
+            if self.container.run_perform_inference(ModelProcessing.DEFAULT, ModelExecutionMode.TRAIN):
                 dataloaders.append((data.train_dataloader(), ModelExecutionMode.TRAIN))  # type: ignore
```

Comment:
Now that the flags are accessible via a method, refactor that into a loop?

Reply:
Done